### ISSUE TITLE
Constantly crashing

### ISSUE THREAD

**Issue by dogmatic69** [zigbee2mqtt/hassio-zigbee2mqtt#390]
### Description of the issue

Since upgrading to 1.28.0-1 yesterday, after a few hours the service crashes. When going to the add on it is not running. Clicking "start" does not start the service.

The only way I've found to get it working again is to reboot the host, restarting ha is not enough. Still, after a while it crashes again.

I have automatic updates enabled so I'm going to assume everything before this version was ok, I'm not certain of the previous version but usually have everything on the lastest version available.

### Addon version

1.28.0-1

### Platform

ha blue + zzh usb stick

### Logs of the issue (if applicable)

Not found anything so far

**Comment by dogmatic69** [comment]
Here is an example. Updated yesterday afternoon, around 6pm it went dead. When the wife started complaining into the lights not working I had a quick look and restart ha, then the host. All worked again great.

This morning with no interactions in terms of updates, restarts etc, became apparent the lights not working again. The docker service was once again not running and host required a reboot to start working again.

![image](https://user-images.githubusercontent.com/94674/193447059-70161cc1-5421-4cd6-9106-33a4554bd768.png)

**Comment by dogmatic69** [comment]
It's happened again and this is the error

```
Adapter disconnected, stopping
Zigbee2MQTT:debug 2022-10-02 18:34:29: Saving state to file /config/zigbee2mqtt/state.json
Zigbee2MQTT:info  2022-10-02 18:34:29: MQTT publish: topic 'zigbee2mqtt/bridge/state', payload 'offline'
Zigbee2MQTT:info  2022-10-02 18:34:29: Disconnecting from MQTT server
Zigbee2MQTT:info  2022-10-02 18:34:29: Stopping zigbee-herdsman...
Zigbee2MQTT:error 2022-10-02 18:34:29: Failed to stop Zigbee2MQTT

```

**Comment by initd0** [comment]
Not sure if related to https://github.com/Koenkk/zigbee2mqtt/issues/14283 but since update to 1.28.0, it doesnt even start up for me.  I had to roll back to 1.27.2 so i can get it working.   Im guessing something is wrong with 1.28.0, would appreciate devs response.

**Comment by xinyingyin** [comment]
I had the same problem, After upgrading to version 1.28.0, the service will automatically stop: Adapter disconnected, stopping

**Comment by scuppasteve** [comment]
I have the same issue, i tried rolling back and that didn't fix it. Below is my herdsman debug. I tried swapping cables and that didnt help. The adapter shows as available. I am using a TI LAUNCHXL-CC26X2R1.

`[14:35:14] INFO: Preparing to start...
[14:35:14] INFO: Socat not enabled
[14:35:15] INFO: Zigbee Herdsman debug logging enabled
[14:35:15] INFO: Starting Zigbee2MQTT...
Zigbee2MQTT:debug 2022-10-03 14:35:17: Loaded state from file /config/zigbee2mqtt/state.json
Zigbee2MQTT:info  2022-10-03 14:35:17: Logging to console and directory: '/config/zigbee2mqtt/log/2022-10-03.14-35-17' filename: log.txt
Zigbee2MQTT:debug 2022-10-03 14:35:17: Removing old log directory '/config/zigbee2mqtt/log/2022-10-03.14-26-20'
Zigbee2MQTT:info  2022-10-03 14:35:17: Starting Zigbee2MQTT version 1.28.0 (commit #unknown)
Zigbee2MQTT:info  2022-10-03 14:35:17: Starting zigbee-herdsman (0.14.62)
Zigbee2MQTT:debug 2022-10-03 14:35:17: Using zigbee-herdsman with settings: '{"adapter":{"concurrent":null,"delay":null,"disableLED":false},"backupPath":"/config/zigbee2mqtt/coordinator_backup.json","databaseBackupPath":"/config/zigbee2mqtt/database.db.backup","databasePath":"/config/zigbee2mqtt/database.db","network":{"channelList":[11],"extendedPanID":[221,221,221,221,221,221,221,221],"networkKey":"HIDDEN","panID":6754},"serialPort":{"path":"/dev/ttyACM1"}}'
2022-10-03T21:35:17.590Z zigbee-herdsman:adapter Failed to validate path: 'Error: spawn udevadm ENOENT'
2022-10-03T21:35:17.591Z zigbee-herdsman:controller:log Starting with options '{"network":{"networkKeyDistribute":false,"networkKey":[1,3,5,7,9,11,13,15,0,2,4,6,8,10,12,13],"panID":6754,"extendedPanID":[221,221,221,221,221,221,221,221],"channelList":[11]},"serialPort":{"path":"/dev/ttyACM1"},"databasePath":"/config/zigbee2mqtt/database.db","databaseBackupPath":"/config/zigbee2mqtt/database.db.backup","backupPath":"/config/zigbee2mqtt/coordinator_backup.json","adapter":{"disableLED":false,"concurrent":null,"delay":null}}'
2022-10-03T21:35:17.592Z zigbee-herdsman:adapter:zStack:znp:log Opening SerialPort with /dev/ttyACM1 and {"baudRate":115200,"rtscts":false,"autoOpen":false}
2022-10-03T21:35:17.596Z zigbee-herdsman:adapter:zStack:znp:log Serialport opened
2022-10-03T21:35:17.597Z zigbee-herdsman:adapter:zStack:znp:SREQ --> SYS - ping - {"capabilities":1}
2022-10-03T21:35:17.598Z zigbee-herdsman:adapter:zStack:unpi:writer --> frame [254,0,33,1,32]
2022-10-03T21:35:17.850Z zigbee-herdsman:adapter:zStack:znp:log Writing CC2530/CC2531 skip bootloader payload
2022-10-03T21:35:17.851Z zigbee-herdsman:adapter:zStack:unpi:writer --> buffer [239]
2022-10-03T21:35:18.853Z zigbee-herdsman:adapter:zStack:znp:SREQ --> SYS - ping - {"capabilities":1}
2022-10-03T21:35:18.853Z zigbee-herdsman:adapter:zStack:unpi:writer --> frame [254,0,33,1,32]
2022-10-03T21:35:19.105Z zigbee-herdsman:adapter:zStack:znp:log Skip bootloader for CC2652/CC1352
2022-10-03T21:35:19.558Z zigbee-herdsman:adapter:zStack:znp:SREQ --> SYS - ping - {"capabilities":1}
2022-10-03T21:35:19.559Z zigbee-herdsman:adapter:zStack:unpi:writer --> frame [254,0,33,1,32]
2022-10-03T21:35:25.561Z zigbee-herdsman:adapter:zStack:znp:SREQ --> SYS - ping - {"capabilities":1}
2022-10-03T21:35:25.563Z zigbee-herdsman:adapter:zStack:unpi:writer --> frame [254,0,33,1,32]
2022-10-03T21:35:31.568Z zigbee-herdsman:adapter:zStack:znp:SREQ --> SYS - ping - {"capabilities":1}
2022-10-03T21:35:31.569Z zigbee-herdsman:adapter:zStack:unpi:writer --> frame [254,0,33,1,32]
Zigbee2MQTT:error 2022-10-03 14:35:37: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2022-10-03 14:35:37: Failed to start zigbee
Zigbee2MQTT:error 2022-10-03 14:35:37: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
Zigbee2MQTT:error 2022-10-03 14:35:37: Exiting...
Zigbee2MQTT:error 2022-10-03 14:35:37: Error: Failed to connect to the adapter (Error: SRSP - SYS - ping after 6000ms)
    at ZStackAdapter.start (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/adapter/zStackAdapter.ts:103:27)
    at Controller.start (/app/node_modules/zigbee-herdsman/src/controller/controller.ts:132:29)
    at Zigbee.start (/app/lib/zigbee.ts:58:27)
    at Controller.start (/app/lib/controller.ts:101:27)
    at start (/app/index.js:109:5)`

**Comment by nashaden** [comment]
Similar problem here. After updating to 1.28.0-1 all my ZigBee routers are marked as offline. When I try to click on any of the routers I get "Bad gateway" error. When I restart the addon they appear again and everything is fine for a few hours, after which they stop working again. I reverted to 1.27.2-1 and the problem was gone. Today I updated to 1.28.0-1 and after 4 hours the routers went offline again (restarting the addon made them come back). I reverted to 1.27.2-1 again and am waiting for a bug fix in 1.28.

My routers are (among others) sonoff zbmini, sonoff basic zb3, tuya ts0002_switch_module and many more.

My coordinator is Sonoff ZBBridge with Tasmota 12.1.1.
